### PR TITLE
RDoc-1895 Add info on an option and setting up 2FA in cloud login article

### PR DESCRIPTION
**Pull request by hiddenshadow21**

### CODE REVIEW

**Review by gregolsky — APPROVED** [review]

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 1]
```diff
@@ -140,6 +140,31 @@ To access your account, click [https://cloud.ravendb.net/user/login](https://clo
...
+---
+
+#### Enable MFA
+
+To enable MFA go to [https://cloud.ravendb.net/portal/profile/enable-authenticator](https://cloud.ravendb.net/portal/profile/enable-authenticator)
+or go to your RavenDB Cloud Profile page ([https://cloud.ravendb.net/portal/profile](https://cloud.ravendb.net/portal/profile))
+and click **Add authenticator app** button.
+!["Add authenticator app"](images\mfa-001-add-authenticator.png "Add authenticator app")
+
+This will take you to configuration page.
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown:154]: configuration --> the configuration

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 2]
```diff
@@ -140,6 +140,31 @@ To access your account, click [https://cloud.ravendb.net/user/login](https://clo
...
+
+!["Configure authenticator app"](images\mfa-002-configuration.png "Configure authenticator app")
+
+- Scan provided QR code or enter text code into authentication app of your choice.
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown:158]: --> Scan the provided QR code or enter the provided text code into an authentication app of your choice.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 3]
```diff
@@ -140,6 +140,31 @@ To access your account, click [https://cloud.ravendb.net/user/login](https://clo
...
+- Type in 2FA code generated by your authentication app.
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown:159]: 2FA code --> the 2FA code

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 4]
```diff
@@ -140,6 +140,31 @@ To access your account, click [https://cloud.ravendb.net/user/login](https://clo
...
+- Type in recovery phone number.
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown:160]: recovery --> a recovery

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 5]
```diff
@@ -140,6 +140,31 @@ To access your account, click [https://cloud.ravendb.net/user/login](https://clo
...
+
+After verifying you will be provided with recovery codes. Save them in secure place.
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown:162]: --> After verifying, you will be provided with recovery codes. Save them in a secure place.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 6]
```diff
@@ -140,6 +140,31 @@ To access your account, click [https://cloud.ravendb.net/user/login](https://clo
...
+{PANEL: Enable MFA}
+
+---
+
+#### Enable MFA
+
+To enable MFA go to [https://cloud.ravendb.net/portal/profile/enable-authenticator](https://cloud.ravendb.net/portal/profile/enable-authenticator)
+or go to your RavenDB Cloud Profile page ([https://cloud.ravendb.net/portal/profile](https://cloud.ravendb.net/portal/profile))
+and click **Add authenticator app** button.
+!["Add authenticator app"](images\mfa-001-add-authenticator.png "Add authenticator app")
```
Comment (reebhub) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown:152]: image

File: Documentation/5.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 7]
```diff
@@ -140,6 +140,31 @@ To access your account, click [https://cloud.ravendb.net/user/login](https://clo
 
 {PANEL/}
 
+{PANEL: Enable MFA}
+
```
Comment (reebhub) [inline Documentation/5.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown:144]: see 4.2 comments above

**Review by reebhub — CHANGES_REQUESTED** [review]

**Review by reebhub — APPROVED** [review]

**Review by gregolsky — APPROVED** [review]

**Pull request merged**